kolibri/plugins/learn/assets/src/views/content-points/index.vue

```diff
@@ -81,6 +84,11 @@
       closePopover() {
         this.internalPopoverShown = false;
       },
+      popOverSetTime(newVal, oldVal) {
+        if (newVal === true && oldVal !== true) {
+          setTimeout(this.closePopover.bind(this), 5000);
```

Comment:
I'm going to recommend removing `.bind`.

As far as I can tell it's not necessary because vue methods are already bound to the vm context